### PR TITLE
Implement Artifact Find Screen

### PR DESCRIPTION
**Pull request by puru-khedre**
### Implement Artifact Find Screen
- Added a search bar to search Artifacts with a query string
- Form list to show the list of the Artifacts and link to there details page

### CODE REVIEW

File: screen/ArtifactLifeCycleApp/ArtifactEvent/FindArtifactEvent.xml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+-->
+<screen xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="http://moqui.org/xsd/xml-screen-3.xsd"
+>
+    <transition name="viewArtifactEvent">
+        <default-response url="../ViewArtifactEvent"/>
+    </transition>
+    <actions>
+        <set field="artifactEventId" value="100408"/>
+<!--        <if condition="!anyField"><set field="anyField" value="*"/></if>-->
```
Comment (puru-khedre) [inline screen/ArtifactLifeCycleApp/ArtifactEvent/FindArtifactEvent.xml]: Remove comment

File: screen/ArtifactLifeCycleApp/ArtifactEvent/FindArtifactEvent.xml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+<http://creativecommons.org/publicdomain/zero/1.0/>.
+-->
+<screen xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="http://moqui.org/xsd/xml-screen-3.xsd"
+>
+    <transition name="viewArtifactEvent">
+        <default-response url="../ViewArtifactEvent"/>
+    </transition>
+    <actions>
+        <set field="artifactEventId" value="100408"/>
```
Comment (puru-khedre) [inline screen/ArtifactLifeCycleApp/ArtifactEvent/FindArtifactEvent.xml]: Useless code, remove

File: screen/ArtifactLifeCycleApp/ArtifactEvent/FindArtifactEvent.xml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+                    <container-dialog id="contentDialog" button-text="Content">
+                        <form-single name="ContentForm">
+                            <field name="content"><default-field title=""><text-line/></default-field></field>
+                        </form-single>
+                    </container-dialog>
+                </default-field>
+            </field>
+
+        </form-list>
+        <field name="artifactEventId">
```
Comment (puru-khedre) [inline screen/ArtifactLifeCycleApp/ArtifactEvent/FindArtifactEvent.xml]: Remove the temporary button

File: screen/ArtifactLifeCycleApp/ArtifactEvent/FindArtifactEvent.xml [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+        xsi:noNamespaceSchemaLocation="http://moqui.org/xsd/xml-screen-3.xsd"
+>
+    <transition name="viewArtifactEvent">
+        <default-response url="../ViewArtifactEvent"/>
+    </transition>
+    <actions>
+        <set field="artifactEventId" value="100408"/>
+<!--        <if condition="!anyField"><set field="anyField" value="*"/></if>-->
+        <set field="queryString" from="anyField ?: '*' "/>
+        <service-call name="co.hotwax.alc.SearchService.search#ArtifactLogIndexes" in-map="[queryString:queryString, pageIndex:pageIndex, pageSize:pageSize, pageNoLimit:pageNoLimit]" out-map="context"/>
```
Comment (puru-khedre) [inline screen/ArtifactLifeCycleApp/ArtifactEvent/FindArtifactEvent.xml]: Fix service name

**Comment by puru-khedre** [comment]
@dt2patel This PR is ready for review

**Review by dt2patel — APPROVED** [review]

**Pull request merged**